github/resource_organization.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package github
 
 import (
+	"fmt"
+
 	"github.com/google/go-github/v32/github"
 
```

Comment:
Unnecessary newline

Reply:
Stefan wanted our internal imports to be separate from external, so I'll hold this for now